Dockerfile

```diff
@@ -4,15 +4,13 @@ MAINTAINER Andy Driver <andy.driver@digital.justice.gov.uk>
 
 WORKDIR /home/cpanel
 
-ADD app app/
-ADD bin bin/
 ADD package.json package.json
-
 RUN npm install
-ADD node_modules node_modules/
+
+ADD app app/
+ADD bin bin/
```

Comment:
I like this - one point: `bin/` will change far less frequently than `app/`, so put it first and we'll benefit from the cache even more

Reply:
Good point, will change 👍